source/blog/2017-05-08-running-and-recording-fully-automated-GUI-tests-in-the-cloud.html.md

```diff
@@ -191,7 +191,7 @@ be lost unless they're uploaded to an external storage.
 
 Fortunately Software Factory handles this: predefined publishers can be appended
 to our jobs definitions; [one of
-which](https://softwarefactory-project.io/r/gitweb?p=config.git;a=blob;f=jobs/_default_jobs.yaml;#l39)
+which](https://softwarefactory-project.io/r/gitweb?p=config.git;a=history;f=jobs/_default_jobs.yaml;h=abedee1d8899afbea7cc1fb31e5b5ee959fba154;hb=04f4d5aa6887f63155ceef38231399a077aa0040)
```

Comment:
Small nit: https://softwarefactory-project.io/r/gitweb?p=config.git;a=blob;f=jobs/_default_jobs.yaml;h=abedee1d8899afbea7cc1fb31e5b5ee959fba154;hb=04f4d5aa6887f63155ceef38231399a077aa0040

To fit with the blog post content, it'd be better to show the file content

Reply:
thanks for the clarification, @hguemar!